oauth2client/contrib/gce.py

```diff
@@ -85,32 +138,128 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
     """
 
-    @util.positional(2)
-    def __init__(self, scope='', **kwargs):
+    def __init__(self,
+                 scope=None,
+                 service_account_email='default',
```

Comment:
Definitely avoid having a non-`None` keyword argument default value, even if it does make the class' internals slightly more complicated.